todo_app/integration_test.py

```diff
@@ -1,84 +1,89 @@
 import pytest
 import dotenv
-from dotenv import load_dotenv,find_dotenv
+from dotenv import load_dotenv, find_dotenv
 import os
 from os.path import join, dirname
 from dotenv import find_dotenv
 from todo_app.app import create_app
-from unittest.mock import patch,Mock
+from unittest.mock import patch, Mock
 #import responses
 import requests
 #import vcr
+import mongomock
+from flask import Flask
+import pymongo
 
 
 @pytest.fixture
 def client():
-# Use our test integration config instead of the'real' version
+    # Use our test integration config instead of the'real' version
     file_path = find_dotenv('.env.test')
     load_dotenv(file_path, override=True)
-# Create the new app.
-#test_app = app.create_app()
-    test_app = create_app()
-# Use the app to create a test_client that can be used in our tests.
-    with test_app.test_client() as client:
-        yield client
+    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
+
+        test_app = create_app()
+        with test_app.test_client() as client:
+            objects = {"name": "test", "idBoard": "test",
+                       "last_modified": "2021-03-15"}
+            db_client = pymongo.MongoClient('fakemongo.com')
+            db_client.db.collection.insert_one(objects)
+            yield client
 
 
 @patch('requests.get')
 def test_index_page(mock_get_requests, client):
-# Replace call to requests.get(url) with our own function
-     mock_get_requests.side_effect = mock_get_lists
-     url = f'/'
-     response = client.get(url)
-     assert "test data" in response.data.decode()
-     
-    
+    # Replace call to requests.get(url) with our own function
+    mock_get_requests.side_effect = mock_get_lists
```

Comment:
This line no longer makes sense either (and indeed errors as the function is no longer defined), we can get rid of this (plus the patch/argument for requests.get)

Reply:
Updated, thank you